static/css/restyle/restyle.less

```diff
@@ -551,7 +551,8 @@ body:not(.home) .amo-header,
   margin-top: 22px;
   padding: 15px 45px 15px 15px;
 
-  &::before, &::after {
+  &::before,
+  &::after {
```

Comment:
I forgot to remove this but it's in keeping with the style of that file... mind if I just leave it in? 😄